### ISSUE TITLE
Just pointing that pip si not finding pervect. it seems that it is not in the PIPY repository

### ISSUE THREAD

**Issue by Maurizio-sanarico-sdg** [scikit-tda/pervect#1]

**Comment by lmcinnes** [comment]
Yes -- we haven't quite got all the pieces in place so it hasn't been uploaded yet. Hopefully in the next month or so it will be generally available. You can still install from github for now, but the README was written in anticipation of an eventual release soon.

**Comment by Maurizio-sanarico-sdg** [comment]
Thank you, it was clear.


--------------------------------------
Maurizio Sanarico
Chief Data Scientist
SDG Group
cell. +39 348 5165701

**Comment by lmcinnes** [comment]
It is now on PyPI. Yay!

**Comment by Maurizio-sanarico-sdg** [comment]
Great.


--------------------------------------
Maurizio Sanarico
Chief Data Scientist
SDG Group
cell. +39 348 5165701
